www/js/services.js

```diff
@@ -24,27 +20,19 @@ angular.module('hall.services', [])
     }
   }
 })
-.factory('Council', function($http, apiUrl) {
-
-  var council = [];
-
+.factory('Council', function(cityApi) {
   return {
     all: function() {
-      return $http.get(apiUrl + '/councillor').then(function(response){
-        council = response.data;
-        return council;
+      return cityApi.people.getAllPeople().then(function(people){
```

Comment:
Super experimental, but we now have a full, human-readable client lib that's autogenerated from the grape API docs :)